app/database/accessor/breeze_buddy/template.py

```
@@ -429,6 +431,73 @@ async def get_template_by_outbound_number_id(
+                shop_identifier=row.get("shop_identifier"),
+                name=row["name"],
+                is_active=row["is_active"],
+                created_at=row["created_at"],
+                updated_at=row["updated_at"],
+            )
+            logger.info(f"Atomic delete succeeded for template: {deleted.id}")
+            return deleted
+        logger.info(
+            f"Atomic delete failed (template in use or not found): {template_id}"
+        )
+        return None
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)